File: sdk/php/src/Model/LoginRequest.php
<?php

namespace HJTPX\Captcha\Model;

class LoginRequest
{
    public $username;
    public $password;
    public $captchaToken;

    public function toArray(): array
    {
        $data = [
            'username' => $this->username,
            'password' => $this->password,
        ];

        if ($this->captchaToken !== null) {
            $data['captcha_token'] = $this->captchaToken;
        }

        return $data;
    }
}

File: sdk/java/src/main/java/com/hjtpx/captcha/model/LoginRequest.java
package com.hjtpx.captcha.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class LoginRequest {
    @JsonProperty("username")
    private String username;

    @JsonProperty("password")
    private String password;

    @JsonProperty("captcha_token")
    private String captchaToken;

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getCaptchaToken() {
        return captchaToken;
    }

    public void setCaptchaToken(String captchaToken) {
        this.captchaToken = captchaToken;
    }
}
